### DISCUSSION TITLE
Hard to find repositories I'm working on

### DISCUSSION THREAD

**Discussion by arsaccol** [community/community#52108] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Whenever I open up GitHub, the first thing I see is a social network feed in a website that I use for daily work, which is not only not the content I'd like to see, it also feels hostile to people like myself who have a hard time keeping focus. I'd rather have some shortcut to the latest repository I've sent contributions to, for instance, with as little superfluous information as possible. Is there such an option by any chance, such as not having a feed at all, or only showing repositories that I've personally touched? GitHub's front page UI may have too much unneeded information, given the hard time I seem to have reading it.

**Comment by adlai** [comment]
Bookmark your own github profile, maybe with the anchor to your own activity; it is not the perfect solution, naturally.

e.g. http://github.com/arsaccol#js-contribution-activity might work for you
